src/fparser/two/symbol_table.py

```diff
@@ -263,6 +263,229 @@ def remove(self, name):
         del self._symbol_tables[lname]
 
 
+class ModuleUse:
```

Comment:
pylint gives the following warnings ...

symbol_table.py:298:29: R1729: Use a generator instead 'all(isinstance(item[0], str) and (item[1] is None or isinstance(item[1], str)) for item in only_list)' (use-a-generator)
symbol_table.py:310:31: R1729: Use a generator instead 'all(isinstance(item[0], str) and isinstance(item[1], str) for item in rename_list)' (use-a-generator)
symbol_table.py:332:30: R1718: Consider using a set comprehension (consider-using-set-comprehension)
symbol_table.py:339:32: R1718: Consider using a set comprehension (consider-using-set-comprehension)
symbol_table.py:366:15: R1729: Use a generator instead 'all(isinstance(item, tuple) and len(item) == 2 for item in tlist)' (use-a-generator)
symbol_table.py:440:41: W0212: Access to a protected member _local_to_module_map of a client class (protected-access)

Reply:
Oops. Fixed now.